docs/public-networks/get-started/manage-memory.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Besu memory management
+---
+
+# Managing Java Virtual Machine (JVM) memory
+
+To setup JVM memory usage for Besu modify maximum heap size using the `JAVA_OPTS` environment variable.
+
+The JVM default is to use 25% of system RAM.
+For example, if you have 16GB RAM installed, the JVM will default to 4GB.
+
+We recommend setting the maximum heap size to at least 8GB for public networks.

```

Comment:
Great contribution @grasponcrypto.
I think we can be more specific here, we recommend on public networks to set -Xmx to 3 GiB on a 8 GiB RAM system, 5 GiB on a 16 GiB system and 8 GiB if you have more than 24 GiB total RAM.
Choosing a higher -Xmx, more than 5 GiB, will speed up the sync period but will not have that much impact after sync, thus we recommend to set -Xmx to 8 GiB when you have available RAM.